javascript/ql/src/semmle/javascript/frameworks/ClosureLibrary.qll

```diff
@@ -0,0 +1,58 @@
+/**
+ * Provides models for miscellaneous utility functions in the closure standard library.
```

Comment:
```suggestion
 * Provides models for functions in the closure standard library.
```
(we have no other occurrences of "miscellaneous" in our docstrings)

Reply:
But there are many functions in the closure library that aren't modelled here. They are in concept-specific libraries like `UriLibraries.qll`, so this really is just the category for things that don't fit anywhere else.